### DISCUSSION TITLE
Mac Mini Asks for Credentials after login from the terminal

### DISCUSSION THREAD

**Discussion by muradabujamous** [community/community#61161] [New to GitHub]
Hello Dears,

We have Mac Mini and we need to use our GitHub account on daily basis.
Recently, after logging in through the terminal to push new commit, we are asked to enter the login information again.

Note: The times for requesting to re-enter the login information are not regular. It may take a day, two days, or a week from the date of the last login 

Please Advice.

**Comment by queenofcorgis** [comment]
👋🏼 I'm checking in with our Account Security team to see if they have any recommendations. I'll report back when I hear from them!

**Reply by queenofcorgis** [reply]
@muradabujamous after discussing with them, this situation may be due to an issue in your configuration so please open a ticket on our [Support page](https://support.github.com/contact). Best of luck!
